koap/js_tests/prepare-testing.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd ../..
+
+./gradlew -Pnpm.publish.repository.github.authToken=fakingit -Pnpm.publish.version=0.0.3-test24 assemble
+
+cd -
+
+npm uninstall @juullabs/koap
+
+npm install file://../build/publications/npm/js
+
+npm run test
```

Comment:
This should become a Gradle task (maybe `tsTest` or similar name) instead of a bash script, so that `check` task can depend on it — and then it will be automatically executed for the current CI setup and under the standard `check` task run.